pkg/util/testkafka/cluster.go

```diff
@@ -62,51 +67,64 @@ func addSupportForConsumerGroups(t testing.TB, cluster *kfake.Cluster, topicName
 		return resp, nil, true
 	})
 
-	cluster.ControlKey(kmsg.OffsetFetch.Int16(), func(request kmsg.Request) (kmsg.Response, error, bool) {
+	cluster.ControlKey(kmsg.OffsetFetch.Int16(), func(kreq kmsg.Request) (kmsg.Response, error, bool) {
```

Comment:
Note to reviewers: the mocked `OffsetFetch` was not correct, because it was returning offset 0 for any partition whose offset was not committed yet. In this PR I'm changing the behaviour to mimic the Kafka one, which returns only committed offsets. If a partition didn't have any committed offset, then the partition will not be present in the response.